client/orm/db_alias.go

```diff
@@ -110,6 +126,20 @@ type DB struct {
 	DB                  *sql.DB
 	stmtDecorators      *lru.Cache
 	stmtDecoratorsLimit int
+
+	Name            string
+	Driver          DriverType
+	DriverName      string
+	DataSource      string
+	MaxIdleConns    int
+	MaxOpenConns    int
+	ConnMaxLifetime time.Duration
+	ConnMaxIdletime time.Duration
+	StmtCacheSize   int
+	//DB              *DB
+	DbBaser dbBaser
+	TZ      *time.Location
+	Engine  string
```

Comment:
can we make these fields as private?

I think developers should not access these fields directly